.github/workflows/reusable/metric.yml

```diff
@@ -0,0 +1,27 @@
+on:
```

Comment:
Since this is a fairly simple action, we can consider using reusable action (formerly called [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action)) instead of reusable workflow. I have an example composite action [here](https://github.com/aws-amplify/amplify-ui/blob/main/.github/actions/install-with-retries/action.yml).

To describe briefly, workflow consists of multiple jobs; and jobs consists of multiple actions (steps). Since this is a fairly simple workflow, imo composite action is better used here.

Reply:
Done!